### PR TITLE
Using env-var 'SENTRY_URL' with 'SENTRY_API_TAGS' and 'SENTRY_API_ISSUES'

### PR DESCRIPTION
**Pull request by dvandra**
Using env-var 'SENTRY_URL' with 'SENTRY_API_TAGS' and 'SENTRY_API_ISSUES' to generate 'sentry_api_tags' and 'sentry_api_issues' URLs.

Adding all 3 env-var to template file.

### CODE REVIEW

File: openshift/template.yaml [hunk 1]
```diff
@@ -40,6 +40,10 @@ objects:
                   value: ${KEEP_WORKER_RESULT_NUM_DAYS}
                 - name: GENERATE_MANIFESTS
                   value: ${GENERATE_MANIFESTS}
+                - name: SENTRY_API_ISSUES
+                  value: ${SENTRY_API_ISSUES}
+                - name: SENTRY_API_TAGS
+                  value: ${SENTRY_API_TAGS}
```
Comment (yzainee) [inline openshift/template.yaml:46]: provide the default macro value in the template as well
example
- description: Dry run
  displayName: Dry run
  required: true
  name: DRY_RUN
  value: "false"

**Comment by deepak1725** [comment]
@dvandra Can you please incorporate these changes too.  
> Sliding Generating Manifests to weekly instead of Daily. 

or this PR:
https://github.com/fabric8-analytics/f8a-stacks-report/pull/147

File: openshift/template.yaml [hunk 2]
```diff
@@ -285,3 +285,15 @@ parameters:
   name: GENERATE_MANIFESTS
   value: "False"
 
+- description: "Genrate sentry-api-issues URL"
```
Comment (tisnik) [inline openshift/template.yaml:288]: 'Generate'

File: openshift/template.yaml [hunk 3]
```diff
@@ -285,3 +285,15 @@ parameters:
...
+  displayName: SENTRY API ISSUES
+  required: true
+  name: SENTRY_API_ISSUES
+  value: "/api/0/projects/sentry/fabric8-analytics-production/issues/"
+
+- description: "Genrate sentry-api-tags URL"
```
Comment (tisnik) [inline openshift/template.yaml:294]: 'Generate'

File: openshift/template.yaml (hunk 2 above)
Comment (dvandra) [reply openshift/template.yaml]: Typo error!

File: openshift/template.yaml (hunk 1 above)
Comment (dvandra) [reply openshift/template.yaml:46]: Done, 
Thanks @yzainee

**Comment by deepak1725** [comment]
@dvandra if I guessed it right, we are embedding `SENTRY_URL` from Secrets in Production template..? I will have a different value for each Environment? stage/ prod..?

**Comment by dvandra** [comment]
> @dvandra if I guessed it right, we are embedding `SENTRY_URL` from Secrets in Production template..? I will have a different value for each Environment? stage/ prod..?

Yes, Deepak it will have diff values for pros and stage.

**Pull request merged**